src/VolumeDrawable.ts

```diff
@@ -685,4 +696,15 @@ export default class VolumeDrawable {
     this.volumeRendering.setRotation(this.rotation);
     this.meshVolume.setRotation(this.rotation);
   }
+
+  setupGui(pane: Pane): void {
+    pane.addInput(this, "translation").on("change", ({ value }) => this.setTranslation(value));
+    pane.addInput(this, "rotation").on("change", ({ value }) => this.setRotation(value));
+
+    const pathtrace = pane.addFolder({ title: "Pathtrace", expanded: false });
+    pathtrace.addInput(this, "primaryRayStepSize").on("change", ({ value }) => this.setRayStepSizes(value));
+    pathtrace
+      .addInput(this, "secondaryRayStepSize")
+      .on("change", ({ value }) => this.setRayStepSizes(undefined, value));
```

Comment:
Do these options have reasonable min/max values?

Reply:
reasonable min is 1.0.  zero is a crashy value, and anything less than 1 would be odd.   Max is something like 20.  I'll double check from agave

Reply:
Agave uses setRange(1.0, 100.0);

Reply:
Basically the values are supposed to be roughly equivalent to voxels and you don't really want rays stepping over that many voxels, so a reasonable max is something like half the expected number of voxels. give or take.